settings.gradle.kts

```diff
@@ -6,16 +6,16 @@ pluginManagement {
     includeBuild("gradle/build-logic")
 
     repositories {
-        gradlePluginPortal()
```

Comment:
Why this change?

Reply:
the mavenCentral() host the most library so I place it first.